test/integration/jest.config.json

```diff
@@ -0,0 +1,16 @@
+{
+	"modulePaths": [
+		"<rootDir>/test/",
+		"<rootDir>/server/",
+		"<rootDir>/client/",
+		"<rootDir>/client/extensions/"
+	],
+	"rootDir": "./../../",
+	"testEnvironment": "node",
+	"testMatch": [
+		"**/bin/**/__integration__/*.js",
+		"**/client/**/__integration__/*.js",
+		"**/server/**/__integration__/*.js"
```

Comment:
Oh interesting it's a Jest naming convention. If we choose to use this style, perhaps we should port over `__test__` in other PRs